source-test/File.ts

```diff
@@ -1,661 +1,1617 @@
+'use strict';
 
-import File = require('vinyl');
-import * as Stream from 'stream';
-import * as fs from 'fs';
+var fs = require('fs');
```

Comment:
Use `import * as fs from 'fs'`, and where not possible, use `import x = require('x');`